frontend/src/components/crud/DisplayTableContainer.tsx

```diff
@@ -103,20 +116,43 @@ const DisplayTable = (props: any) => {
   );
 };
 
+// graphql {
+const ENTITIES = gql`
+  {
```

Comment:
One thing I've seen from working with graphql before is explicitly naming all queries and mutations like eg.
```
const ENTITIES = gql` 
  query DisplayTableContainer_Entities {
```
I think it's so if graphql throws an error, we can track where the error occurred more easily

Reply:
Thanks, good to know! I'll do that 👍